docs/console/console-extensibility/overview.md

```diff
@@ -0,0 +1,28 @@
+---
+id: overview
+title: Overview
+sidebar_label: Overview
+---
+# Overview
+
+The Extensibility feature can make your work easier and allow you to integrate your external tools via iframes. In this way the Console will be able to incorporate more features from other websites (or from the Console projects itself) and thus become a single access point making the user experience more comfortable.
+
+A correctly active extension will appear as a new menu item on the sidebar and you can place it in different Console contexts such as within a Company or a specific Project. Furthermore, you can also specify restrictions using Console permissions so that the new menu item is visible only to a small group of users.
```

Comment:
```suggestion
A correctly activated extension will appear as a new menu item on the Console main navigation. You can place extensions in different Console contexts such as within a Company or inside a specific Project. Furthermore, you can also specify restrictions using Console permissions so that the new menu item is visible only to a limited group of users.
```